app/controllers/business_entities_controller.rb

```diff
@@ -0,0 +1,110 @@
+class BusinessEntitiesController < ApplicationController
+  def index
+    authorize :business_entity
+    list_jurisdictions
+  end
+
+  def new
+    redirect_to office_business_entities_path if valid_business_entity || !jurisdiction
+    @business_entity = BusinessEntity.new
+    authorize @business_entity
+  end
+
+  def create
+    @business_entity = business_entity_service.build(business_entity_params)
+    authorize @business_entity
+    if @business_entity.save
+      redirect_to office_business_entities_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    authorize business_entity
+  end
+
+  def update
+    bes = business_entity_service
+    new_be = bes.check_update(business_entity_params)
+    authorize new_be
+
+    if bes.persist_update!(new_be)
+      redirect_to office_business_entities_path
```

Comment:
Personally, I'd avoid the abbreviations of the variables and I'd call them by what they are.

Also, do we need to have **bes** variable at all? Could this:

```ruby
def update
  bes = business_entity_service
  new_be = bes.check_update(business_entity_params)
  authorize new_be

  if bes.persist_update!(new_be)
```
be written as:

```ruby
def update
  new_business_entity = business_entity_service.check_update(business_entity_params)
  authorize new_business_entity

  if business_entity_service.persist_update!(new_business_entity)
```

The reason I'm asking to avoid the abbreviated variable names is because I think the second example 'reads' easier. There is no need for the 'translation' as to what the abbreviations stand for.

What do you think?